src/nfcsettings.cpp

```diff
@@ -43,17 +43,17 @@ NfcSettings::NfcSettings(QObject *parent)
     , m_valid(false)
     , m_enabled(false)
     , m_available(false)
-    , m_interface(new NemoDBus::Interface(
+    , m_interface(
             this, QDBusConnection::systemBus(),
```

Comment:
If `NemoDBus::Interface` is a `QObject` (I think it is) and the first parameter (`this`) is its parent then I would rather allocate it dynamically. I'm not sure that Qt parent <-> child machinery works reliably for `QObject`'s allocated on stack or as part of another object.

Reply:
It's not. It's "Object" without the Q, which is a class internal to the module.